modules/manage/pages/kubernetes/k-manage-topics.adoc

```diff
@@ -565,7 +565,7 @@ include::develop:partial$topic-defaults.adoc[]
 
 == Suggested reading
 
-- xref:reference:k-topic-crd.adoc[]
+- xref:reference:k--crd.adoc[]
```

Comment:
Should this be `k--crd.adoc` or `k-crd.adoc`? Single `-` or double?

Reply:
thanks!